canvas.go

```diff
@@ -23,6 +21,12 @@ const (
 	Lossy
 )
 
+// PixelsPerMillimeter (PPM) for the resolution of raster images. Higher DPM will result in bigger images.
+type PixelsPerMillimeter float64
+
+// DPI is a shortcut for Dots-per-Inch for the resolution of raster images.
+const DPI = PixelsPerMillimeter(1 / 25.4)
```

Comment:
Should be `25.4` not the reciprocal, i.e. a DPI of `72.0` equals a DPMM of `2.8`, so that `2.8 [dots-per-mm] * 25.4 [mm/in] = 72.0 [in]